radixdlt/src/main/java/org/radix/GlobalInjector.java

```diff
@@ -158,6 +160,20 @@ public RadixAddress get() {
 		}
 	}
 
+	static class SelfBFTNodeProvider implements Provider<BFTNode> {
```

Comment:
I think this could be converted to a provider method, given that is what we are using elsewhere?